natlas-server/app/static/natlas.js

```diff
@@ -8,6 +8,7 @@ import { registerTagModalEvents } from './controls/natlas-tagging';
 import { registerAgentEvents } from './controls/user-profile';
 import 'natlas.scss';
 import 'bootstrap';
+require("particles.js")
```

Comment:
`import "particles.js"`

Reply:
Doesn't work :-/

Reply:
Why not? It should work. They're the same thing